go-controller/pkg/config/config.go

```diff
@@ -1208,19 +1225,41 @@ var MasterHAFlags = []cli.Flag{
 	},
 	&cli.IntFlag{
 		Name:        "ha-election-renew-deadline",
-		Usage:       "Leader election renew deadline (in secs) (default: 35)",
+		Usage:       "Leader election renew deadline (in secs) (default: 30)",
```

Comment:
We can use `Sprintf` so we don't harcode default string description here.

Also, why do we need different defaults ?